smelt.lic

```diff
@@ -91,10 +101,23 @@ class Smelt
   def fuel
     pause 1
     waitrt?
-    get_crafting_item('shovel', @belt)
-    bput('push fuel with my shovel', 'roundtime')
+    if @tool == 'tong'
+      get_crafting_item(@tool, @belt)
+      bput("adjust my #{@tool}","You lock")
+    else
+      stow_crafting_item(@tool, @bag, @belt)
```

Comment:
Why stow `@tool` only to get it again immediately?

The same question applies to lines 118 and 119.